### ISSUE TITLE
ID Attribute Accessibility

### ISSUE THREAD

**Issue by jolanglinais** [accordproject/template-studio-v2#41]
# WCAG 2.0 Level A [4.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html) (Parsing)

## Tasks
- All `id` attributes should only be used once.

## Related Issues: 
- Issue [#56](https://github.com/accordproject/cicero-ui/issues/56) in Cicero-UI

**Comment by monika** [comment]
Hi! I audited template studio with [axe](https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) and as far as I can see, the multiple `id` conflicts are from the svg icons in the text editing toolbar and the trash icon when deleting a clause.

I'm thinking, like the issue stated, fixing the issue in https://github.com/accordproject/cicero-ui/issues/56 should fix this one?

<img width="1490" alt="Screen Shot 2019-10-05 at 7 01 52 PM" src="https://user-images.githubusercontent.com/316762/66263149-9e6a5e00-e7a2-11e9-8fdf-9c656e6d8925.png">

**Comment by jolanglinais** [comment]
Hello @monika! This makes sense, `template-studio-v2` should be comprised primarily of components from `cicero-ui`, so that issue will likely be where most of the work stems from.
